capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -1902,4 +1912,28 @@ private void onBackPressed() {
             mCancelListener.onCancelFlow();
         }
     }
+
+    private void trackCameraNoPermissionMessageShownEvent() {
```

Comment:
[Suggestion] I think we don't need to add functions for one line actions. What do you think? If we need them, I would suggest better names. This is the CameraAccess permission screen, not the NoPermissionMessage ....

Reply:
Resolved